src/imago.js

```diff
@@ -112,17 +147,44 @@
         height: self.elements.figure.clientHeight,
       };
 
-      document.addEventListener('mousemove', crop);
-      document.addEventListener('mouseup', stopCropping);
+      if(e.type == 'touchend') {
+        // Touch Event
+        self.tmp.mouse = {
+          x: e.changedTouches[0].pageX,
+          y: e.changedTouches[0].pageY,
+        };
+
+        document.addEventListener('touchmove', crop);
+        document.addEventListener('touchend', stopCropping);
+
```

Comment:
nice video :vhs: but did you try to crop the image using your fingers? maybe the crop handlers are two small for cropping, increase handlers size to see if works :santa: